filament/backend/src/opengl/PlatformEGL.cpp

```diff
@@ -166,19 +173,19 @@ Driver* PlatformEGL::createDriver(void* sharedContext) noexcept {
         goto error;
     }
 
-    if (configsCount == 0) {
-      // warn and retry without EGL_RECORDABLE_ANDROID
+    if (recordable == 1 && configsCount == 0) {
+        // warn and retry without EGL_RECORDABLE_ANDROID
         logEglError("eglChooseConfig(..., EGL_RECORDABLE_ANDROID) failed. Continuing without it.");
-      // this is not fatal
-      configAttribs[12] = EGL_RECORDABLE_ANDROID;
-      configAttribs[13] = EGL_DONT_CARE;
-      if (!eglChooseConfig(mEGLDisplay, configAttribs, &mEGLTransparentConfig, 1, &configsCount) ||
-              configsCount == 0) {
-          logEglError("eglChooseConfig");
-          goto error;
-      }
+        // this is not fatal
+        configAttribs[12] = EGL_RECORDABLE_ANDROID;
+        configAttribs[13] = EGL_DONT_CARE;
+        if (!eglChooseConfig(mEGLDisplay, configAttribs, &mEGLTransparentConfig, 1, &configsCount) ||
+                configsCount == 0) {
+            logEglError("eglChooseConfig");
+            goto error;
+        }
     }
-
+    
```

Comment:
👾

Reply:
Did Visual Studio do this? 🤨

Reply:
https://github.com/shapr3d/filament/pull/134/commits/54f4490633762fdb2c6a2dff13ebe0c7caf83a48